{"name": "Авторизация несуществующего пользователя", "status": "failed", "statusDetails": {"message": "AssertionError: assert (200 == 404)", "trace": "self = <couriers.test_courier.TestCourier object at 0x0000025BFD143E30>\n\n    @allure.title(\"Авторизация несуществующего пользователя\")\n    def test_autorisation_no_exist_user(self):\n        courier = CourierMethods()\n        response, status_code = courier.courier_login_no_exist_data(COURIER_DATA_NO_EXIST_ACCOUNT)\n>       assert status_code == 404 and response == {\"code\": 404, \"message\": \"Учетная запись не найдена\"}\nE       assert (200 == 404)\n\ntests\\couriers\\test_courier.py:94: AssertionError"}, "steps": [{"name": "Авторизация несуществующего курьера", "status": "passed", "parameters": [{"name": "params", "value": "{'login': 'no_exist_login', 'password': 'no_exist_password', 'firstname': 'no_exist_firsname'}"}], "start": 1750370280200, "stop": 1750370280304}], "start": 1750370280200, "stop": 1750370280304, "uuid": "be9f3b1f-6bfa-4af2-8759-8a812bac64f8", "historyId": "6d038a1508d343547aa8690e0fe9e8cb", "testCaseId": "6d038a1508d343547aa8690e0fe9e8cb", "fullName": "tests.couriers.test_courier.TestCourier#test_autorisation_no_exist_user", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "DESKTOP-04L8HEE"}, {"name": "thread", "value": "15132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_courier"}]}